tools/mixxx-isolate.py

```diff
@@ -0,0 +1,496 @@
+        return True
+    # move processes until empty
+    processes = 1
+    while processes:
+        try:
+            log.debug(" Move processes from cpuset %s %s", setInstance, name)
+            proc.move(setInstance, "/", None, False)
+            processes = setInstance.tasks
+            log.debug(" Processes left: %s", processes)
+            break
+        except BrokenPipeError:
+            continue
+        except Exception as e:
+            log.exception(e)
+            print(e)
+
+    ccset.destroy(name)
+    return True
+
+
+def teardown(args, isolate):
+    cset.CpuSet(None)
+
+    if not args.no_offline:
+        if args.cleanup:
+            # since our detection algorithm can't detect isolation cpus
+            # form offline CPUs, we will just bring up all cores on teardown
+            for fname in os.listdir("/sys/devices/system/cpu"):
+                try:
+                    if re.match("cpu[0-9]+", fname):
+                        with open(
+                            "/sys/devices/system/cpu/%s/online" % fname, "w"
+                        ) as fp:
+                            fp.write("1")
+                except OSError:
+                    pass
+                except Exception as e:
+                    print(e)
+
+        for iso in isolate:
+            with open("/sys/devices/system/cpu/cpu%d/online" % iso, "w") as fp:
+                fp.write("1")
+
+    log.info(" Deactivate cpusets")
+    deleteCpuset(args.cpuset_engine, args.debug)
+    deleteCpuset(args.cpuset_isolation, args.debug)
+    deleteCpuset(args.cpuset_system, args.debug)
+
+    if prev_gov is not None:
+        log.info(" Restore governor %s" % prev_gov)
+        system("sudo cpufreq-set -g %s" % prev_gov)
+
+    log.info(" Cleanup finished")
+    print("STATUS: CLEAN")
+
+
+def createcpuset(name, cpuspec, memspec, cx, mx):
+    try:
+        ccset.create(name, cpuspec, memspec, cx, mx)
+    except CpusetExists:
+        pass
+    except Exception as e:
+        raise e
+
+
+def joinCpuIds(lst):
+    return ",".join((str(x) for x in lst))
+
+
+def isolateSystem(args):
+    conf = selectCpus(args.cpu)
+
+    memspec = "0"
+    log.info(" Create cpuset groups")
+    try:
+        createcpuset(
+            args.cpuset_engine, str(conf["use"]), memspec, False, False
+        )
+        createcpuset(
+            args.cpuset_system,
+            joinCpuIds(conf["system"]),
+            memspec,
+            False,
+            False,
+        )
+        if args.isolate and conf["isolate"]:
+            createcpuset(
+                args.cpuset_isolation,
+                joinCpuIds(conf["isolate"]),
+                memspec,
+                False,
+                False,
+            )
+    except Exception as instance:
+        teardown(args, conf["isolate"])
+        # unroll
+        log.exception(instance)
+        log.critical(" Failed to create shield, hint: do other cpusets exist?")
+        sys.exit(23)
```

Comment:
```suggestion
    except:
        teardown(args, conf["isolate"])
        # unroll
        log.exception("Failed to create shield, hint: do other cpusets exist?")
        sys.exit(23)
```

[log.exception](https://docs.python.org/3/library/logging.html#logging.Logger.exception) should already log the current exception details/traceback, and the argument should be used to pass a custom message